### DISCUSSION TITLE
Cant get avatar and push notification to work

### DISCUSSION THREAD

**Discussion by bioblatt** [tigase/siskin-im#153] [Q&A]
I've got the following error if I want to enable push-notification on Siskin:
_It was not possible to contact push notification component: undefined-condition_

And for uploading an avatar:
_User avatar publication failed. Reason: remote-server-timeout_

Prosody 0.11.10-1
iOS 15.0.2
Siskin 6.4

All needed modules for Prosody are enabled.
Push notifications and setting avatars works with Conversations on Android.

**Comment by woj-tek** [comment]
can you open s2s communication from your server to `push.tigase.im`?

**Reply by bioblatt** [reply]
How do I check this?

**Comment by woj-tek** [comment]
1) what is the domain of your server (I could try checking from our side)
2) in any client with service discovery (mostly desktop ones) you could try querying our servers (tigase.im or push.tigase.im)
3) if you are admin of your server you could check prosody logs for any errors regarding connectivity.
